static/redfish/v1/JsonSchemas/index.json

```diff
@@ -403,4 +403,4 @@
       "@odata.id": "/redfish/v1/JsonSchemas/redfish-schema"
     }
   ]
-}
```

Comment:
Why delete it and add it again?

Reply:
I don't know why it is showing really.. but , everything seems be OK to me..